### DISCUSSION TITLE
Node-windows fork?

### DISCUSSION THREAD

**Discussion by carlpaten** [coreybutler/node-windows#292] [Q&A] [answered]
It seems that @coreybutler went on hiatus. Node-windows is a really useful library for us, but there are a couple security issues pending ([here](https://github.com/coreybutler/node-windows/issues/290), [here](https://github.com/coreybutler/node-windows/issues/273)) and perhaps a few improvements that can be made (e.g. support non-global installation somehow, improve error messages when not installed globally).

I'm wondering whether someone has already undertaken to fork and maintain the project. I'd consider writing a couple PRs but I can't actually maintain the project myself.

**Comment by coreybutler** [comment, marked as answer]
Not on hiatus, but definitely backed up. There are some challenges. One of the main goals of this library is to have a unified API across this project, https://github.com/coreybutler/node-mac, and https://github.com/coreybutler/node-mac.

The first issue mentioned here is a bot-generated issue. There is a PR #267 to replace minimist. It looks good (and leverages another library I wrote, reducing maintenance burdens), but hasn't been tested. If folks have a chance to test that, it would help.

Several updates have been made, but many were rolled back, causing some issues on some Windows builds. If the problems can be narrowed down to more than just "it doesn't work", more PR's can be merged. I have a limited number of environments to test on (mostly Windows Pro). I don't have access to many versions of Windows server (licenses).
